File: problemas/iniciante/1114/1114.cs
using System;

class URI {
    static void Main(string[] args) {
        string linha = string.Empty;

        while ((linha = Console.ReadLine()) != null){
            int senha = int.Parse(linha);

            if(senha == 2002){
                Console.WriteLine("Acesso Permitido");
                break;
            }

            Console.WriteLine("Senha Invalida");
        }
    }
}

File: problemas/iniciante/1114/1114.dart
import 'dart:io';

void main() {
  while (true) {
    int n = int.parse(stdin.readLineSync()!);

    if (n == 2002) {
      print('Acesso Permitido');
      break;
    }

    print('Senha Invalida');
  }
}
